Add Arch Linux installation instructions for hyprshot (#139)
Gustash/Hyprshot: README.md

> You can install the [hyprshot](https://aur.archlinux.org/packages/hyprshot) package in AUR.

Reviewer: ```suggestion
You can install the [hyprshot](https://archlinux.org/packages/extra/any/hyprshot/) package from the extra repo.
```
